crates/biome_js_formatter/tests/specs/prettier/js/babel-plugins/deferred-import-evaluation.js.snap

```diff
@@ -1,47 +0,0 @@
----
-source: crates/biome_formatter_test/src/snapshot_builder.rs
-info: js/babel-plugins/deferred-import-evaluation.js
----
-
-# Input
-
-```js
-import defer * as ns from "x";
-
-```
-
-
-# Prettier differences
-
-```diff
---- Prettier
-+++ Biome
-@@ -1 +1,2 @@
--import defer * as ns from "x";
-+import defer
-+* as ns from "x"
-```
-
-# Output
-
-```js
-import defer
-* as ns from "x"
-```
-
-# Errors
-```
-deferred-import-evaluation.js:1:14 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
-
-  × expected `from` but instead found `*`
-  
-  > 1 │ import defer * as ns from "x";
```

Comment:
This is no longer reported because this is recovered as a combiner import clause (`defer` is the default specifier). Not sure why it didn't error asking for a comma?